setup.py

```diff
@@ -54,6 +54,7 @@
         'phonopy==2.20.0',
         'pint==0.22',
         'pyiron_base==0.6.7',
+        'pyiron-data==0.0.22',
```

Comment:
There is no `pyiron-data` package on PYPI.

Reply:
Right, I guess it's not exactly .py code is it